assets/html/index.html

```diff
@@ -1 +1,22 @@
-
+<html>
+  <head>
+    <title>Challenge One</title>
+  </head>
+  <body>
+    <h1 align="center">This is Chris Featley's page</h1>
```

Comment:
don't use inline styles. check out how you can link a css file. https://www.w3schools.com/css/css_howto.asp

Reply:
have linked a css file